### ISSUE TITLE
implement `zz`

### ISSUE THREAD

**Issue by ghost** [aioutecism/amVim-for-VSCode#40]
````
The following changes since commit dc564bd6180744fe9cf203994e1aae124010c58c:

  Update version to 0.1.1. (2016-02-10 07:22:51 +0900)

are available in the git repository at:

  https://gitlab.com/crussell/amVim.git zz

for you to fetch changes up to f5d2263d292926bf54060542c73e2325784ca17f:

  Implement `zz` (2016-02-11 21:51:21 -0600)

----------------------------------------------------------------
Colby Russell (1):
      Implement `zz`

 src/Actions/Reveal.ts | 11 +++++++----
 src/Modes/Normal.ts   |  3 +++
 2 files changed, 10 insertions(+), 4 deletions(-)
````

Merge this using `git pull https://gitlab.com/crussell/amVim.git zz`

**Comment by aioutecism** [comment]
Commented https://gitlab.com/crussell/amVim/commit/f5d2263d292926bf54060542c73e2325784ca17f#note_3682229

**Comment by colbyrussell** [comment]
Address review comments: https://gitlab.com/crussell/amVim/commit/c565aa4ab9dd7d59456719b2ea70bab773e2390e

**Comment by aioutecism** [comment]
Merged 7b50f0e.

**Comment by aioutecism** [comment]
I checked #1 and found `z.` should does the same thing as `zz`.
Added in 2853d93.
